src/components/postCompletionViews/StartPaymentView.stories.js

```diff
@@ -0,0 +1,35 @@
+import {BASE_URL} from 'api-mocks';
+import {
+  mockSubmissionPaymentStartGet,
+  mockSubmissionProcessingStatusGet,
+} from 'api-mocks/submissions';
+import {withSubmissionPollInfo} from 'story-utils/decorators';
+
+import {StartPaymentViewDisplay} from './StartPaymentView';
+
+export default {
+  title: 'Private API / Post completion views / Start payment',
+  render: StartPaymentViewDisplay,
```

Comment:
```suggestion
  title: 'Private API / Post completion views / Start payment',
  component: StartPaymentViewDisplay,
```
should have effectively the same behaviour, but the meta-data tracking about the component is improved in storybook